test/test_utils.py

```diff
@@ -68,3 +69,11 @@ def set_rng_seed(seed):
 def get_asset_path(file_name: str) -> str:
     """Get the path to the file under assets directory."""
     return str(_ASSET_DIR.joinpath(file_name))
+
+
+def assert_tensors_equal(actual: Tensor, expected: Tensor):
```

Comment:
Allow optional arguments so the caller can access `atol` `rtol`.
Just name it `assert_expected` as we may want to check both cases of strict equality and closeness.

For example: https://github.com/pytorch/vision/blob/main/test/test_models.py#L97